dali/test/python/operator_2/test_resize.py

```diff
@@ -803,6 +804,45 @@ def test_very_small_output():
             yield _test_very_small_output, dim, device
 
 
+large_data = None
+large_data_resized = None
+
+
+@params((types.INTERP_NN, False), (types.INTERP_LINEAR, False), (types.INTERP_LINEAR, True))
+def test_large_gpu(interp, antialias):
```

Comment:
how costly is it? Should we mark it as slow or something?

Reply:
Not very. A few seconds.